sycl/test/warnings/sycl_2020_deprecations.cpp

```diff
@@ -134,10 +134,10 @@ int main() {
   sycl::byte B;
   (void)B;
 
-  // expected-warning@+1{{'max_constant_buffer_size' is deprecated: max_constant_buffer_size is deprecated}}
+  // expected-error@+1{{no member named 'max_constant_buffer_size' in 'sycl::info::device'}}
   auto MCBS = sycl::info::device::max_constant_buffer_size;
   (void)MCBS;
-  // expected-warning@+1{{'max_constant_args' is deprecated: max_constant_args is deprecated}}
+  // expected-error@+1{{no member named 'max_constant_args' in 'sycl::info::device'}}
   auto MCA = sycl::info::device::max_constant_args;
   (void)MCA;
 
```

Comment:
I suggest we just remove these checks from the tests.

Reply:
It was decided to add 2 checks for removed API : check that API is not available without this macro and check that API still working with this macro. This is check for compile error

Reply:
> I think first of these two checks must be here

Why? What is the value in checks that functionality is not supported?

Reply:
This API was not removed from the code, but was wrapped in the macro. If code of API exists, we should check that it is unavailable

Reply:
> If code of API exists, we should check that it is unavailable

I disagree with the way it's implemented. I don't see any value in checking that some functionality is not supported regardless whether it's code removed from sources or by pre-processor.

I withdraw my request for changes, but I'd like @s-kanaev, @vladimirlaz or @romanovvlad approve before it goes in because I see inconsistency in testing removed features.